test/parallel/test-runner-output.mjs

```diff
@@ -2,6 +2,7 @@ import * as common from '../common/index.mjs';
 import * as fixtures from '../common/fixtures.mjs';
 import * as snapshot from '../common/assertSnapshot.js';
 import { describe, it } from 'node:test';
+import { replaceWindowsPaths } from '../common/assertSnapshot.js';
```

Comment:
any reason for importing this instead of calling `snapshot.replaceWindowsPaths`? (line 3 imports it with `* as snapshot`

Reply:
nope, missed that 😅